docker-compose.yaml

```diff
@@ -84,10 +96,43 @@ services:
         depends_on:
             - db
             - mq
+        networks:
+            - daisy_network
         command: "celery -A elixir_daisy beat --loglevel=DEBUG --pidfile= --scheduler django_celery_beat.schedulers:DatabaseScheduler"
-
+    # Backup service
+    backup:
+        image: alpine:3.20.3
+        environment:
+            - DB_HOST=db
+            - DB_PORT=5432
+            - DB_NAME=${DB_NAME:-daisy}
+            - DB_USER=${DB_USER:-daisy}
+            - DB_PASSWORD=${DB_PASSWORD:-daisy}
+            - BACKUP_DIR=/backups
```

Comment:
can we specify this in the env file? 
Can this be an external folder? (outside of the container)

Reply:
`BACKUP_DIR` refers to the directory inside the container. To find the corresponding directory on the host VM, check the volume mounted at `$BACKUP_VOLUME`.